src/components/BounceRules/index.jsx

```diff
@@ -0,0 +1,233 @@
+import React from "react";
+import { Redirect } from "react-router-dom";
+import "./BounceRules.css";
+import axios from "axios";
+
+export default class BounceRules extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      searchToken: "",
+      isRedirectingToDetail: false,
+      selectedRule: {},
+      rules: [],
+      pageIndex: 1,
+      pageInterval: 10,
+    };
+
+    this.updateSearchToken = this.updateSearchToken.bind(this);
+    this.handleKeyDown = this.handleKeyDown.bind(this);
+  }
+
+  async componentDidMount() {
+    const { data, status } = await axios.get(
+      "http://localhost:3000/bounce_rules"
```

Comment:
We can configure some environment variable/some base URL across different environments rather than hardcoding http://localhost:3000 so we can eventually do `${envSpecificBaseUrl}/bounce_rules`

Reply:
consider webpack defineplugin or something like that to configure the base url based on different environments

Reply:
resolved with .env file